api/test/wfl/integration/modules/wgs_test.clj

```diff
@@ -36,21 +37,32 @@
                    "Inputs are not at the top-level"))]
          (run! check-nesting (:workflows workload))))))
 
-(defn ^:private old-create-wgs-workload! []
-  (let [request (make-wgs-workload-request)]
-    (jdbc/with-db-transaction [tx (fixtures/testing-db-config)]
-      (let [[uuid table] (all/add-workload-table! tx wgs/workflow-wdl request)
-            add-id (fn [m id] (assoc (:inputs m) :id id))]
-        (jdbc/insert-multi! tx table (map add-id (:items request) (range)))
-        (jdbc/update! tx :workload {:version "0.3.8"} ["uuid = ?" uuid])
-        uuid))))
+(defn ^:private old-create-wgs-workload!
+  ([] (old-create-wgs-workload! nil))
+  ([workflow-options]
+   (let [request (make-wgs-workload-request)]
+     (jdbc/with-db-transaction [tx (fixtures/testing-db-config)]
+       (let [[id table] (all/add-workload-table! tx wgs/workflow-wdl request)
+             to-row (fn [m id] (assoc (:inputs m)
+                                 :id id
+                                 :workflow_options (when workflow-options (json/write-str workflow-options))))]
```

Comment:
The old table didn't store workflow options (and I don't think it should be updated). Can you just add these to the loader?

Reply:
Fair enough. The default loader used for the old database already supports this out of the box (and doesn't complain if it is `nil`) so I've just removed the testing capability for it